cpp/src/io/comp/uncomp.cpp

```diff
@@ -263,7 +263,7 @@ int cpu_inflate(uint8_t *uncomp_data, size_t *destLen, const uint8_t *comp_data,
 }
 
 
-gdf_error io_uncompress_single_h2d(const void *src, gdf_size_type src_size, void **dst_dev, gdf_size_type *dst_size, int strm_type)
+gdf_error io_uncompress_single_h2d(const void *src, gdf_size_type src_size, void **dst, gdf_size_type *dst_size, int strm_type)
```

Comment:
This function needs Doxygen documentation. Everything in this file does, but since you're changing this function, might as well add documentation as well. Boy Scout's Rule: leave it better than you found it :) 

Example template for Doxygen documentation here: https://github.com/rapidsai/cudf/blob/24939c7592be3093c5fd213a0d6c91ba6c8c1cb9/cpp/src/examples/documentation_example.cpp